libs/blocks/marquee-anchors/marquee-anchors.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Marquee Anchors
+ */
+import { createTag, getConfig } from '../../utils/utils.js';
+import { decorateBlockText, getBlockSize, decorateBlockBg } from '../../utils/decorate.js';
+
+// size: [heading, body, ...detail]
+const blockTypeSizes = {
+  default: {
+    xsmall: ['xs', 'xs', 'xs'],
+    small: ['m', 's', 's'],
+    medium: ['l', 'm', 'm'],
+    large: ['xl', 'm', 'l'],
+    xlarge: ['xxl', 'l', 'xl'],
+  },
+};
```

Comment:
Looks like there is a whole system of sizes being used in this block and elsewhere. Might be a good to create a discussion for it.